docker/docker-compose.dev.yml

```diff
@@ -55,7 +55,7 @@ services:
       - ../datahub-frontend/build/stage/playBinary:/datahub-frontend
 
   datahub-upgrade:
-    image: linkedin/datahub-upgrade:debug
+    image: acryldata/datahub-upgrade:debug
```

Comment:
Always uses the _latest_ acryl debug image?

Reply:
These debug images are never created and pushed. They stay local. If you look at the gms one it is like a year old and many of the containers lag a debug tag at all. Therefore these debug images are local only and are never pushed.